dbms/src/Storages/Transaction/tests/kvstore_helper.h

```diff
@@ -123,9 +126,9 @@ class RegionKVStoreTest : public ::testing::Test
         }
         String path = TiFlashTestEnv::getContext().getPath();
         auto p = path + "/metadata/";
-        TiFlashTestEnv::tryRemovePath(p, /*recreate=*/true);
+        TiFlashTestEnv::tryCreatePath(p);
```

Comment:
Why don't we clean up the existing files that the previous run may leave?

Reply:
When we run two test cases in one process, the old logic will remove the folders generated by previous cases. But since the Context instance is already created, the removed folders won't be created again, which causes exceptions by the following logic.
And seems not removing those folders won't cause problems.